### PR TITLE
VZ-8551: VPO status Init:CrashLoopBackOff after applying operator.yaml

### PR DESCRIPTION
**Pull request by pshirodkar-orcl**
This is a follow-up to PR #5732. This is for the automated testing to use any combination of install & upgrade using either kubectl or vz cli.

### CODE REVIEW

File: ci/upgrade-paths/Jenkinsfile [hunk 1]
```diff
@@ -12,6 +12,13 @@ def dns_zone_ocid = 'dummy'
 def OKE_CLUSTER_PREFIX = ""
 def agentLabel = env.JOB_NAME.contains('master') ? "phx-large" : "large"
 
+def KUBECTL = "kubectl"
+def VZ_CLI = "vz cli"
```
Comment (mgianatagh) [inline ci/upgrade-paths/Jenkinsfile:16]: Does this ever get used?

Comment (pshirodkar-orcl) [reply ci/upgrade-paths/Jenkinsfile:16]: Yes, those are used in the two lists defined below.

File: ci/upgrade-paths/Jenkinsfile [hunk 2]
```diff
@@ -920,8 +948,30 @@ def installVerrazzanoOnCluster(count, verrazzanoConfig) {
...
+                    cd ${GO_REPO_PATH}/verrazzano
+                    kubectl -n verrazzano-install rollout status deployment/verrazzano-platform-operator
+
+                    echo "Applying \$verrazzanoConfig"
+                    # validate is set to false since we are using newer Verrazzano CR yaml when installing an older pre-upgrade version
+                    kubectl apply --validate=false -f ${verrazzanoConfig}
+
+                    # wait for Verrazzano install to complete
+                    ./tests/e2e/config/scripts/wait-for-verrazzano-install.sh
+                elif [ "${params.INSTALLATION_METHOD_PREUPGRADE}" == "vz cli" ]; then
```
Comment (mgianatagh) [inline ci/upgrade-paths/Jenkinsfile:971]: I feel it would be better to use the VZ_CLI constant instead of "vz cli".

Comment (pshirodkar-orcl) [reply ci/upgrade-paths/Jenkinsfile:971]: I called it "vz cli" since it shows up a dropdown value in the associated Jenkins paramter.

Comment (mgianatagh) [reply ci/upgrade-paths/Jenkinsfile:971]: I just feel it would better to have == "${VZ_CLI)" instead of repeatedly having references to "vz cli"

Comment (pshirodkar-orcl) [reply ci/upgrade-paths/Jenkinsfile:971]: `VZ_CLI` is a groovy variable. ${VZ_CLI) does not evaluated when referenced in the bash section. So I resorted to hard-coding the string instead.

Comment (mgianatagh) [reply ci/upgrade-paths/Jenkinsfile:971]: ok

**Review by mgianatagh — APPROVED** [review]

**Pull request merged**